fe/fe-core/src/main/jflex/sql_scanner.flex

```diff
@@ -108,6 +108,7 @@ import com.starrocks.qe.SqlModeHelper;
         keywordMap.put("backup", new Integer(SqlParserSymbols.KW_BACKUP));
         keywordMap.put("begin", new Integer(SqlParserSymbols.KW_BEGIN));
         keywordMap.put("between", new Integer(SqlParserSymbols.KW_BETWEEN));
+        keywordMap.put("be_host", new Integer(SqlParserSymbols.KW_BE_HOST));
```

Comment:
Why not `host`? Could other node(front-end, compute node, cache node) share the same keyword?